src/js/components/Paragraph/doc.js

```diff
@@ -29,3 +29,76 @@ export const doc = Paragraph => {
 
   return DocumentedParagraph;
 };
+
+export const themeDoc = {
+  'global.colors.text': {
+    description: 'The text color used inside the Paragraph.',
+    type: 'string | { dark: string, light: string }',
+    defaultValue: "{ dark: '#f8f8f8', light: '#444444' }",
+  },
+  'global.text': {
+    description: 'The text size of the Paragraph text.',
+    type: 'string',
+    defaultValue: 'medium',
+  },
+  'global.edgeSize': {
+    description: 'The possible sizes for margin.',
+    type: 'object',
+    defaultValue: `{
+      edgeSize: {
```

Comment:
I don't think we need to repeat `edgeSize` here, as that's already in the key name.